client/packages/host/package.json

```diff
@@ -20,10 +20,10 @@
     "webpack-dev-server": "^4.8.1"
   },
   "scripts": {
-    "start-remote": "webpack-cli serve",
-    "start-local": "webpack-cli serve --env local",
-    "build": "webpack --env production",
-    "build-local": "webpack --env production local",
+    "start": "webpack-cli serve",
+    "start-remote": "webpack-cli serve --env API_HOST='https://demo-open.msupply.org'",
+    "start-local": "webpack-cli serve --env API_HOST='http://localhost:8000'",
+    "build": "yarn clean && webpack --env production",
```

Comment:
just noticed this - there's no clean script and this fails to build.
I've added back in
```
     "clean": "rimraf dist",
```
is this what you intended? I remember in another issue that you wanted to retain `dist`

Reply:
oops, thanks:
issue: https://github.com/openmsupply/open-msupply/issues/64
PR: https://github.com/openmsupply/open-msupply/pull/65